socorro/processor/rules/mozilla.py

```diff
@@ -8,6 +8,7 @@
 import re
 import time
 from urllib.parse import unquote_plus
+from zlib import error as ZlibError
```

Comment:
For some reason, the exception class is `error` which is unhelpful so I named it something better.